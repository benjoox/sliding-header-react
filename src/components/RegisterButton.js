import React from 'react';
import { ReactComponent as Profile } from './assets/profile.svg'
import '../index.css';

export default function RegisterButton() {
  return (
    <button className="registeration-btn">
      <div>
        <Profile />
      </div>
      <p>Register</p>
    </button>
  )
}