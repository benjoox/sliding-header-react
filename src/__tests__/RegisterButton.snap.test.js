import React from 'react'
import renderer from 'react-test-renderer'
import RegisterButton from '../components/RegisterButton'

it('RegisterButton  renders correctly', () => {
  const tree = renderer.create(<RegisterButton />).toJSON()
  expect(tree).toMatchSnapshot()
})
