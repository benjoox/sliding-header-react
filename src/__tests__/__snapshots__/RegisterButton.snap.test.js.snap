// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterButton  renders correctly 1`] = `
<button
  className="registeration-btn"
>
  <div>
    <svg>
      profile.svg
    </svg>
  </div>
  <p>
    Register
  </p>
</button>
`;
